app/views/projects/show.html.erb

```diff
@@ -10,6 +10,20 @@ Project Details:
   </div>
 <% end %>
 
+<% if !@project.in_mediaflux? %>
+  <div class="alert alert-warning" role="alert">
+    <dl>
+    <dt>Status</dt> <dd><%=@project_status%></dd>
+    </dl>
+  </div>
+<% else @project.in_mediaflux? %>
```

Comment:
It can be just `<% else %>` - you don't need the call to `@project.in_mediaflux?` here.

Reply:
Ahh, correcting now - thanks!